spec/services/moment_keywords_spec.rb

```diff
@@ -0,0 +1,45 @@
+
+describe MomentKeywords do
+    let(:user) { create(:user) }
+    let(:categories) {[FactoryBot.build(:category,
+                                        name: "category name",
+                                        description: "category description."
+    )]}
+    let(:moods) {[FactoryBot.build(:mood,
+        name: "mood-name",
+        description: "mood-description."
+    )]}
+    let(:strategies) {[FactoryBot.build(:strategy,
+        name: "strategy-name",
+        description: "strategy-description."
+    )]}
+    subject(:moment) {FactoryBot.build(:moment,
+                                       name: "Test ADDICTION",
+                                       why: "More testing content self-care.",
+                                       fix: "Text tested is @Teachers!!",
+                                       categories: categories,
+                                       moods: moods,
+                                       strategies: strategies
+                                       ) }
+    let(:keywords) { MomentKeywords.new(moment).extract_keywords }
```

Comment:
This is the class being tested, so you can use subject here

```suggestion
    subject(:keywords) { MomentKeywords.new(moment).extract_keywords }
```